deployment/legacy-migration/maintenance-worker/migration/neo4j/dbManipulations/20191011_changeReportNodeToRelationOnly.cql

```diff
@@ -0,0 +1,7 @@
+MATCH (submitter:User)-[:REPORTED]->(report:Report)-[:REPORTED]->(resource)
+DETACH DELETE report
+CREATE (submitter)-[reported:REPORTED]->(resource)
+SET reported.createdAt = toString(datetime())
+SET reported.reasonCategory = 'other'
+SET reported.reasonDescription = '!!! Created automatically to ensure database consistency! Date-time is this creation date and time.'
+RETURN reported
```

Comment:
this didn't work for me... I get a `Permission denied` error...
it works if I use `cypher-shell` directly, which is probably what I would do

Reply:
I would love to have this minimum temporarily in the code base. For transparency and may be as a conversion example for later generations?

But for sure must not be in the legacy-migration. I couldn’t find quicke a better place in the short time … I like to ask and than forgot.
Do you have a suggestion where to put it? @mattwr18 

It only works in the cypher-shell. Correct. Where you have tried this out?

Reply:
I tried running it from the command line, which can't be done since it requires `cypher-shell`

I think it would be good to convert it to a script, if we want it in the code base.
see `neo4j/db_setup.sh`, it's part of the database and it's manipulation, so it seems like a good place for it.
You can copy and paste a lot of the code from `neo4j/ds_setup.sh`, It is meant to be used once or twice, and it cannot be called directly from the remote server, so we'd still need to open the file locally and copy and paste it. I like that you linked to the example of where you found the way to do it, maybe we don't need an example in our code base, if there are docs with them?